src/content/components/Tooltip.js

```diff
@@ -22,22 +22,22 @@ export default class Tooltip extends PureComponent {
 
   state: {
     interiorElement: HTMLElement | null,
-    interiorElementRAF: HTMLElement | null,
+    isNewContentLaidOut: boolean,
```

Comment:
I refactored this a bit, it still has the same logic as before, but with hopefully better words and comments.